setup.cfg

```diff
@@ -0,0 +1,48 @@
+[wheel]
+universal = 1
+
+[metadata]
+name = polyply
+author = Fabian Grunewald
+author_email = f.grunewald@rug.nl
+description_file = README.md
+description-content-type = text/markdown; charset=UTF-8
+url = https://github.com/marrink-lab/polyply_1.0
+license = Apache 2.0
+classifier =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Scientific/Engineering :: Bio-Informatics
+    Topic :: Scientific/Engineering :: Chemistry
+keywords = molecular dynamics charmm gromos martini polymers
+
+[options]
+packages = find:
+setup-requires =
+    setuptools >= 30.3.0
+    pbr
+install-requires =  # ?? requires-dist?
+    pbr
+    numpy
+    networkx ~= 2.0
+    vermouth
+zip-safe = False
+
+[options.extras_require]
+full = 
+    scipy
```

Comment:
Isn't this a hard dependency?

Reply:
How do you mean?

Reply:
In that it is always required to run polyply. There's no way to run polyply without having scipy installed. If so, move it to install-requires